cask.el

```diff
@@ -392,6 +412,19 @@ SCOPE may be nil or 'development."
                     (epl-invalid-package
                      (cask--show-package-error err filename)))))
              (cask--from-epl-package bundle package))))
+        (package-descriptor
+         (cl-destructuring-bind (_ &optional filename) form
+           (let* ((descriptor-filename
+                   (or filename (let ((pkg-files (f-glob "*-pkg.el" (s-chop-suffix "/" (cask-bundle-path bundle)))))
```

Comment:
`s-chop-suffix` seems unnecessary here. If it were necessary, it seems more likely that `directory-file-name` should be used.

Reply:
I've made this change locally and will update the PR, but having trouble getting tests going on Emacs 26. I'll merge like this and if you have trouble on Emacs > 25 we can look at it afterwards.